packages/builder/src/fee/FeeBuilder.interfaces.ts

```diff
@@ -1,12 +1,28 @@
-import { ChainAssetId } from '@moonbeam-network/xcm-types';
+import { AnyParachain, ChainAssetId } from '@moonbeam-network/xcm-types';
 import { ApiPromise } from '@polkadot/api';
+import { Enum } from '@polkadot/types';
+import { StagingXcmV3MultiLocation } from '@polkadot/types/lookup';
 import { SubstrateCallConfig } from '../types/substrate/SubstrateCallConfig';
 
 export interface FeeConfigBuilder {
   build: (params: FeeConfigBuilderPrams) => SubstrateCallConfig;
 }
 
 export interface FeeConfigBuilderPrams {
+  address: string;
   asset: ChainAssetId;
   api: ApiPromise;
+  transferAsset: ChainAssetId;
```

Comment:
What is `transferAsset`?

Reply:
Is the asset that is being transferred, which in some cases is different that the feeAsset. Since we are in the Fee builder I left the name `asset` for the fee asset.

Reply:
TBH I'm not sure if I improved naming for FeeBuilder in v3 but if not we definitely should.